### DISCUSSION TITLE
IFCConvert: How to split the carcass of a building from an IFC into an STEP or IFC

### DISCUSSION THREAD

**Discussion by Mannshoch** [IfcOpenShell/IfcOpenShell#2896] [Q&A]
I play around with the shell-command but it seems I'm not able to get the what I want. I currently try to not convert the whole IFC into a STEP anymore. I try to get a specific Model part. 

If I export the ifc to an XML, I get these Information about Walls:
```

<IfcBuildingElementProxy id="1ekWTba5nCLhrVLwlcW3lK" Name="Wand-051" Description="" ObjectPlacement="-0.98162717860265891 -0.1908090203019337 0 0 0.1908090203019337 -0.98162717860265891 0 0 0 0 1 0 -5.5043596073199996 17.509540221999998 11.220000000000001 1" Tag="68BA0765-905C-4C56-B91F-57AEE6803BD4">
							<IfcBuildingElementPart id="1q3oaVq3aQETAt1i8ceDSb" Name="Luft 300" ObjectPlacement="-0.98162717860265891 -0.1908090203019337 0 0 0.1908090203019337 -0.98162717860265891 0 0 0 0 1 0 -5.5043596073199996 17.509540221999998 11.220000000000001 1">
								<IfcPresentationLayerAssignment xlink:href="#AR_Brüstungselemente "/>
								<IfcMaterial xlink:href="#IfcMaterial_5433"/>
							</IfcBuildingElementPart>
...
```

I'm not so Into IC but It seems that all Wall Items have the leading part in the name "Wand-*" in common

Does some on know how to get the carcass (all walls) of the building into a STEP or an IFC?

**Comment by aothms** [comment]
There are various inclusion filters in IfcConvert, such as `IfcConvert a.ifc a.stp --include entities IfcWall` or `IfcConvert a.ifc a.stp --include attribute Name Wand-*`.
